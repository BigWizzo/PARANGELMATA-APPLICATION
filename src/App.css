.btn {
  background-color: #094c8e;
  border-color: #4c6180;
}
