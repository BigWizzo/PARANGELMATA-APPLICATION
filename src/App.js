import React, { useState } from 'react';
import { Line } from 'react-chartjs-2';
import './App.css';

function App() {
  const [nation, setNation] = useState('South Africa');
  const data = {
    labels: ['Red', 'Blue', 'Yellow'],
    datasets: [
      {
        label: '# of Votes',
        data: [12, 19, 31],
        backgroundColor: ['rgba(255, 99, 132, 0.2)'],
        borderColor: ['rgba(255, 99, 132, 1)'],
        borderWidth: 1,
      },
    ],
  };
  console.log(data);

  const loading = true;

  return (
    <div className="App">
      <h1>Parangelmata</h1>
      <input type="text" onChange={(e) => setNation(e.target.value)} />
      {loading && <p>Loading...</p>}
      <div className="chart-wrapper">
        <Line data={data} />
      </div>
    </div>
  );
}

export default App;
