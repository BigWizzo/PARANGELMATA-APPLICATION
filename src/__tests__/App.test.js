import React from 'react';
import renderer from 'react-test-renderer';
import App from '../App';
import { Provider } from 'react-redux';
import { store, persistor } from '../redux/store';
import { PersistGate } from 'redux-persist/integration/react';

test('should render the Parangelmata Application Page', () => {
  const component = renderer.create(
    <Provider store={store}>
      <PersistGate loading={null} persistor={persistor}>
        <App />
      </PersistGate>
    </Provider>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
