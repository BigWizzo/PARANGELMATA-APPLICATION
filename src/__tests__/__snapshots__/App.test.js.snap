// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Parangelmata Application Page 1`] = `
<div
  className="App text-light"
>
  <div
    className="main"
  >
    <h1
      className="py-4 text-center"
    >
      Parangelmata
    </h1>
    <form
      className="country-form m-auto mb-5"
      onSubmit={[Function]}
    >
      <div
        className="my-form mb-3"
      >
        <input
          className="form-control"
          onChange={[Function]}
          placeholder="Enter Country"
          type="text"
          value=""
        />
      </div>
      <div
        className="d-grid gap-2 d-block"
      >
        <button
          className="btn btn-primary font-weight-bolder"
          type="submit"
          value="Submit"
        >
          Submit
        </button>
      </div>
    </form>
    <div
      className="content"
    >
      <div
        className="empty text-center"
      >
        <h4>
          Please enter Country Name and hit Submit
        </h4>
      </div>
    </div>
    <div
      class="circle-small-2 br-50 pos-abs"
    />
    <div
      class="circle-small-4 br-50 pos-abs"
    />
    <div
      class="circle-small-5 br-50 pos-abs"
    />
  </div>
</div>
`;
